src/app/home/home.component.spec.ts

```diff
@@ -1,14 +1,18 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { NbDialogService } from '@nebular/theme';
 
 import { HomeComponent } from './home.component';
 
+class MockDialogService { }
+
 describe('HomeComponent', () => {
   let component: HomeComponent;
   let fixture: ComponentFixture<HomeComponent>;
 
-  beforeEach(async(() => {
+  beforeEach(waitForAsync(() => {
     TestBed.configureTestingModule({
-      declarations: [ HomeComponent ]
+      declarations: [ HomeComponent ],
+      providers: [ {provide: NbDialogService, useClass: MockDialogService} ]
```

Comment:
Not sure why the test suite works as the `HomeComponent` uses card and button components. But let's explicitly import required modules.
```suggestion
      imports: [
        CommonModule,
        NbThemeModule.forRoot(),
        NbCardModule,
        NbButtonModule,
        NbDialogModule.forRoot(),
      ],
```